### PR TITLE
N35: Star Machine

### PR DESCRIPTION
**Pull request by Seldom-SE**
_Part of #1065_

Thanks!

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/035/index.md [hunk 1]
```diff
@@ -164,6 +164,19 @@ Interested in contributing? Head over to the
...
 [botnet_ideas]: https://github.com/JMS55/botnet/discussions/categories/ideas
 
+### [Star Machine]
+
+![Screenshot of a level in Star Machine](star_machine.png)
+
+[Star Machine] by [@Seldom_SE] is a puzzle game built in Bevy, where
+you wire together components to escape each level.
+
+The game, although its development is presently inactive, made
```
Comment (ozkriff) [inline content/news/035/index.md:174]: this sentence looks a little weird (sorry for the late review)

Comment (Seldom-SE) [reply content/news/035/index.md:174]: Which sentence specifically? I'm new to GitHub reviews haha

Comment (ozkriff) [reply content/news/035/index.md:174]: "The game <...> made a video demo <...>"

Comment (17cupsofcoffee) [reply content/news/035/index.md:174]: I'll make note to tweak this in the release PR.

**Pull request merged**